velox/functions/sparksql/String.h

```diff
@@ -91,6 +91,28 @@ struct AsciiFunction {
   }
 };
 
+template <typename T>
+struct BitLengthVarcharFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<int32_t>& result,
+      const arg_type<Varchar>& input) {
+    result = input.size() * 8;
+  }
+};
+
+template <typename T>
+struct BitLengthVarbinaryFunction {
```

Comment:
I think we can unify the two functions into just one, named `BitLengthFunction`, by using template type for call method's input. Then, register it for both varchar & varbinary.

See reference:
https://github.com/facebookincubator/velox/blob/main/velox/functions/lib/CheckedArithmetic.h#L28.

Reply:
Updated, thanks!